[BUG] Chip-cert-bins docker script failing on TOT

### Reproduction steps

Running this script is failing on TOT : 

docker buildx build --build-arg COMMITHASH=a55718b6f00f648e72c56316e4d968f929f9cefa --platform linux/arm64 --tag connectedhomeip/chip-cert-bins:a55718b6f00f648e72c56316e4d968f929f9cefa --push .



### Bug prevalence

Everytime

### GitHub hash of the SDK that was being used

a55718b6f00f648e72c56316e4d968f929f9cefa

### Platform

raspi

### Platform Version(s)

linux/arm64

### Anything else?

The docker script is failing again on TOT (a55718b6f00f648e72c56316e4d968f929f9cefa)

```
Failing here : 

#0 1.756 [92/2080] stamp obj/src/controller/tests/TestCommissionableNodeController_generate_driver.stamp
#0 1.758 [93/2080] ACTION //examples/all-clusters-app/all-clusters-common:all-clusters-common_zapgen_zap_pregen(//build/toolchain/linux:linux_arm64_gcc)
#0 1.758 FAILED: gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated/access.h gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated/gen_config.h gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated/endpoint_config.h gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated/CHIPClientCallbacks.h gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated/CHIPClusters.h gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated/IMClusterCommandHandler.cpp 
#0 1.759 python3 ../../third_party/pigweed/repo/pw_build/py/pw_build/python_runner.py --gn-root ../../ --current-path ../../examples/all-clusters-app/all-clusters-common --default-toolchain=//build/toolchain/linux:linux_arm64_gcc --current-toolchain=//build/toolchain/linux:linux_arm64_gcc --capture-output --python-dep-list-files gen/examples/all-clusters-app/all-clusters-common/all-clusters-common_zapgen_zap_pregen_metadata_path_list.txt -- ../../scripts/tools/zap/generate.py --no-prettify-output --templates /root/connectedhomeip/src/app/zap-templates/app-templates.json --output-dir /root/connectedhomeip/out/debug/gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated --lock-file /root/connectedhomeip/out/debug/zap_gen.lock --parallel /root/connectedhomeip/examples/all-clusters-app/all-clusters-common/all-clusters-app.zap
#0 1.759 🚀 Executing: git log -1 --format={"hash": "%H","timestamp": %ct}
#0 1.759 🔍 Git commit: ffd5670eace603e75e150f9eff08e8aed5e1545e from Tue Jan 31 2023 16:46:01 GMT+0000 (Coordinated Universal Time)
#0 1.759 🔍 Current src hash: /1X7+qJiYOFjNhVVwU91NPmcAxs=
#0 1.759 🔍 Current src-shared hash: //f+yOV4+MNjwnqqpFCZgz3pjuU=
#0 1.759 👎 Error reading old hash file: /opt/zap-v2023.01.31-nightly/spa/hash.json
#0 1.759 🐝 Front-end code changed, so we need to rebuild SPA.
#0 1.759 🚀 Executing: npx quasar build
#0 1.759 ⇝ 
#0 1.759 --------------------------------------------------------
#0 1.759  INCOMPATIBLE NODE VERSION
#0 1.759  @quasar/app-webpack requires Node 12.22.1 or superior
#0 1.759 
#0 1.759  You are running Node v12.19.0
#0 1.759  Please install a compatible Node version and try again
#0 1.759 --------------------------------------------------------
#0 1.759 
#0 1.759 👎 Program npx exited with error code: 1
#0 1.759 1
#0 1.759 Searching for zcl file from /root/connectedhomeip/examples/all-clusters-app/all-clusters-common/all-clusters-app.zap
#0 1.759 Unable to check version using %r. Assuming version is ok. ['node', 'src-script/zap-start.js']
#0 1.759 Traceback (most recent call last):
#0 1.759   File "/root/connectedhomeip/out/debug/../../scripts/tools/zap/generate.py", line 345, in <module>
#0 1.759     main()
#0 1.759   File "/root/connectedhomeip/out/debug/../../scripts/tools/zap/generate.py", line 322, in main
#0 1.759     runGeneration(cmdLineArgs)
#0 1.759   File "/root/connectedhomeip/out/debug/../../scripts/tools/zap/generate.py", line 214, in runGeneration
#0 1.759     tool.run('generate', *args)
#0 1.759   File "/root/connectedhomeip/scripts/tools/zap/zap_execution.py", line 109, in run
#0 1.759     subprocess.check_call(
#0 1.759   File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
#0 1.759     raise CalledProcessError(retcode, cmd)
#0 1.759 subprocess.CalledProcessError: Command '['node', 'src-script/zap-start.js', 'generate', '-z', '/root/connectedhomeip/src/app/zap-templates/zcl/zcl-with-test-extensions.json', '-g', '/root/connectedhomeip/src/app/zap-templates/app-templates.json', '-i', '/root/connectedhomeip/examples/all-clusters-app/all-clusters-common/all-clusters-app.zap', '-o', '/root/connectedhomeip/out/debug/gen/examples/all-clusters-app/all-clusters-common/zap_pregen/zap-generated', '--tempState']' returned non-zero exit status 1.
#0 1.774 [94/2080] ACTION //src/controller/tests:TestEventChunking_generate_driver(//build/toolchain/linux:linux_arm64_gcc)
#0 1.782 [95/2080] ACTION //src/controller/tests:TestEventNumberCaching_generate_driver(//build/toolchain/linux:linux_arm64_gcc)
#0 1.782 [96/2080] ACTION //src/controller/tests:TestReadChunking_generate_driver(//build/toolchain/linux:linux_arm64_gcc)
#0 1.905 [97/2080] ACTION //src/controller/data_model:data_model_zapgen_zap_pregen(//build/toolchain/linux:linux_arm64_gcc)
#0 1.905 FAILED: gen/src/controller/data_model/zap_pregen/zap-generated/access.h gen/src/controller/data_model/zap_pregen/zap-generated/gen_config.h gen/src/controller/data_model/zap_pregen/zap-generated/endpoint_config.h gen/src/controller/data_model/zap_pregen/zap-generated/CHIPClientCallbacks.h gen/src/controller/data_model/zap_pregen/zap-generated/CHIPClusters.h gen/src/controller/data_model/zap_pregen/zap-generated/IMClusterCommandHandler.cpp 
#0 1.905 python3 ../../third_party/pigweed/repo/pw_build/py/pw_build/python_runner.py --gn-root ../../ --current-path ../../src/controller/data_model --default-toolchain=//build/toolchain/linux:linux_arm64_gcc --current-toolchain=//build/toolchain/linux:linux_arm64_gcc --capture-output --python-dep-list-files gen/src/controller/data_model/data_model_zapgen_zap_pregen_metadata_path_list.txt -- ../../scripts/tools/zap/generate.py --no-prettify-output --templates /root/connectedhomeip/src/app/zap-templates/app-templates.json --output-dir /root/connectedhomeip/out/debug/gen/src/controller/data_model/zap_pregen/zap-generated --lock-file /root/connectedhomeip/out/debug/zap_gen.lock --parallel /root/connectedhomeip/src/controller/data_model/controller-clusters.zap
#0 1.905 🚀 Executing: git log -1 --format={"hash": "%H","timestamp": %ct}
#0 1.905 🔍 Git commit: ffd5670eace603e75e150f9eff08e8aed5e1545e from Tue Jan 31 2023 16:46:01 GMT+0000 (Coordinated Universal Time)
#0 1.905 🔍 Current src hash: /1X7+qJiYOFjNhVVwU91NPmcAxs=
#0 1.905 🔍 Current src-shared hash: //f+yOV4+MNjwnqqpFCZgz3pjuU=
#0 1.905 👎 Error reading old hash file: /opt/zap-v2023.01.31-nightly/spa/hash.json
#0 1.905 🐝 Front-end code changed, so we need to rebuild SPA.
#0 1.905 🚀 Executing: npx quasar build
#0 1.905 ⇝ 
#0 1.905 ⇝ --------------------------------------------------------
#0 1.905  INCOMPATIBLE NODE VERSION
#0 1.905  @quasar/app-webpack requires Node 12.22.1 or superior
#0 1.905 
#0 1.905  You are running Node v12.19.0
#0 1.905 ⇝  Please install a compatible Node version and try again
#0 1.905 --------------------------------------------------------
#0 1.905 
#0 1.905 👎 Program npx exited with error code: 1
#0 1.905 1
#0 1.905 Searching for zcl file from /root/connectedhomeip/src/controller/data_model/controller-clusters.zap
#0 1.905 Unable to check version using %r. Assuming version is ok. ['node', 'src-script/zap-start.js']
#0 1.905 Traceback (most recent call last):
#0 1.905   File "/root/connectedhomeip/out/debug/../../scripts/tools/zap/generate.py", line 345, in <module>
#0 1.905     main()
#0 1.905   File "/root/connectedhomeip/out/debug/../../scripts/tools/zap/generate.py", line 322, in main
#0 1.905     runGeneration(cmdLineArgs)
#0 1.905   File "/root/connectedhomeip/out/debug/../../scripts/tools/zap/generate.py", line 214, in runGeneration
#0 1.905     tool.run('generate', *args)
#0 1.905   File "/root/connectedhomeip/scripts/tools/zap/zap_execution.py", line 109, in run
#0 1.905     subprocess.check_call(
#0 1.905   File "/usr/lib/python3.10/subprocess.py", line 369, in check_call
#0 1.905     raise CalledProcessError(retcode, cmd)
#0 1.905 subprocess.CalledProcessError: Command '['node', 'src-script/zap-start.js', 'generate', '-z', '/root/connectedhomeip/src/app/zap-templates/zcl/zcl.json', '-g', '/root/connectedhomeip/src/app/zap-templates/app-templates.json', '-i', '/root/connectedhomeip/src/controller/data_model/controller-clusters.zap', '-o', '/root/connectedhomeip/out/debug/gen/src/controller/data_model/zap_pregen/zap-generated', '--tempState']' returned non-zero exit status 1.
#0 2.022 [98/2080] ACTION //src/controller/data_model:data_model_codegen_codegen(//build/toolchain/linux:linux_arm64_gcc)
#0 2.022 ninja: build stopped: subcommand failed.
------
Dockerfile:206
--------------------
 205 |     SHELL ["/bin/bash", "-c"]
 206 | >>> RUN set -x && \
 207 | >>>     source scripts/activate.sh \
 208 | >>>     && gn gen out/debug --args='chip_mdns="platform" chip_inet_config_enable_ipv4=false' \
 209 | >>>     && ninja -C out/debug
 210 |     
--------------------
ERROR: failed to solve: process "/bin/bash -c set -x &&     source scripts/activate.sh     && gn gen out/debug --args='chip_mdns=\"platform\" chip_inet_config_enable_ipv4=false'     && ninja -C out/debug" did not complete successfully: exit code: 1
testingmatters@TestingMatterss-MacBook-Pro chip-cert-bins % 
```

Also attaching the full log 
[Docker_build_failure.zip](https://github.com/project-chip/connectedhomeip/files/10581529/Docker_build_failure.zip)

--- a/integrations/docker/images/chip-cert-bins/Dockerfile
+++ b/integrations/docker/images/chip-cert-bins/Dockerfile
@@ -163,10 +163,10 @@ RUN case ${TARGETPLATFORM} in \
             set -x \
             && mkdir node_js \
             && cd node_js \
-            && wget https://nodejs.org/dist/v12.19.0/node-v12.19.0-linux-x64.tar.xz \
-            && tar xfvJ node-v12.19.0-linux-x64.tar.xz \
-            && mv node-v12.19.0-linux-x64 /opt/ \
-            && ln -s /opt/node-v12.19.0-linux-x64 /opt/node \
+            && wget https://nodejs.org/dist/v12.22.12/node-v12.22.12-linux-x64.tar.xz \
+            && tar xfvJ node-v12.22.12-linux-x64.tar.xz \
+            && mv node-v12.22.12-linux-x64 /opt/ \
+            && ln -s /opt/node-v12.22.12-linux-x64 /opt/node \
             && ln -s /opt/node/bin/* /usr/bin \
             && cd .. \
             && rm -rf node_js \
@@ -175,10 +175,10 @@ RUN case ${TARGETPLATFORM} in \
             set -x \
             && mkdir node_js \
             && cd node_js \
-            && wget https://nodejs.org/dist/v12.19.0/node-v12.19.0-linux-arm64.tar.xz \
-            && tar xfvJ node-v12.19.0-linux-arm64.tar.xz \
-            && mv node-v12.19.0-linux-arm64 /opt/ \
-            && ln -s /opt/node-v12.19.0-linux-arm64 /opt/node \
+            && wget https://nodejs.org/dist/v12.22.12/node-v12.22.12-linux-arm64.tar.xz \
+            && tar xfvJ node-v12.22.12-linux-arm64.tar.xz \
+            && mv node-v12.22.12-linux-arm64 /opt/ \
+            && ln -s /opt/node-v12.22.12-linux-arm64 /opt/node \
             && ln -s /opt/node/bin/* /usr/bin \
             && cd .. \
             && rm -rf node_js \

